app/src/main/java/com/lagradost/cloudstream3/syncproviders/providers/MALApi.kt

```diff
@@ -445,6 +449,12 @@ class MALApi(index: Int) : AccountManager(index), SyncAPI {
                 null,
                 null,
                 plot = this.node.synopsis,
+                releaseDate = if (this.node.start_date == null) null else Date.from(
+                    Instant.from(
```

Comment:
I dont trust this to not throw an exception as `Throws: DateTimeException – if unable to convert to an Instant` 

Wrap this in a try catch